common/lib/protocol-definitions/package.json

```diff
@@ -28,48 +54,38 @@
 		"eslint": "eslint --format stylish src",
 		"eslint:fix": "eslint --format stylish src --fix --fix-type problem,suggestion,layout",
 		"format": "npm run prettier:fix",
-		"lint": "npm run prettier && npm run check:release-tags && npm run eslint",
+		"lint": "npm run prettier && npm run check:release-tags && npm run eslint && npm run check:are-the-types-wrong",
```

Comment:
Unless things have changed, `npm run eslint` needs to appear last in the lint scripts (at least for packages that fluid-build interacts with), because arguments are passed down for it from fluid-build. Has that changed?

Reply:
This package is not configured to build using fluid-build.